awx/ui/src/components/Schedule/shared/DateTimePicker.js

```diff
@@ -42,19 +46,23 @@ function DateTimePicker({ dateFieldName, timeFieldName, label }) {
           ? 'default'
           : 'error'
       }
-      label={`${label} date/time`}
+      label={`${label}`}
     >
       <DateTimeGroup>
         <DatePicker
-          aria-label={t`${label} date`}
+          aria-label={
+            dateFieldName.startsWith('start') ? t`Start date` : t`End date`
```

Comment:
Same comment as above about dateFieldName.

Reply:
It is a string, that could possibly change, but then the form won't work because that string is the same name as the values key.  I am unable to simply grab the key off of formik and use that in the `DateTimePicker` because there are cases where there are 2 of the components on the screen at the same time that manipulate Start date/time and End date/time of the form. Instead, I'll pass the name of the field into `<DateTimePicker>` based on the appropriate component name (End date/time vs Start date/time).  This way, if the field name changes in formik it'll still work.